rules_csharp_gapic/csharp_gapic_pkg.bzl

```diff
@@ -64,7 +64,7 @@ tar -czhpf {out_tar} -C {out_dir}/.. {pkg_name}
         inputs =  [gapic_zip] + extras
         command = """
 {zipper} x {gapic_zip} -d {out_dir}
-CLIENT_NAME=$(ls -1 {out_dir} | sort | head -n 1)
+CLIENT_NAME=$(grep libraryPackage {out_dir}/gapic_metadata.json | head -n 1 | cut -d'"' -f 4)
```

Comment:
Please add the note from the CL as a comment here for clarity

Reply:
Added as a new commit for simplicity. I'll squash them into one commit before merging.